src/components/ComponentDemo/Code/CodeBar.js

```diff
@@ -17,21 +17,24 @@ const StorybookLink = ({ framework, url }) => (
   </a>
 );
 
-const CodeBar = ({ src, code, links }) => {
+const CodeBar = ({ src, code, links, showSandboxURL }) => {
   const sandboxUrl = useCodesandbox(code);
   const storybookLinks = Object.entries(links);
 
```

Comment:
**subjective alert** Do you mind moving this into a single, affirmative boolean. Just to clarify the intent a bit we can avoid chaining shortcuts. Not sure how you would have known this, it should be codified in an eslint rule.

```suggestion
const shouldShowCopyButton = !src && code;

```